src/shared/DMIParser.c

```diff
@@ -0,0 +1,100 @@
+/*
+ * Created by Aavild on 19/11/2021.
+ */
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <cJSON.h>
+#include "DMIParser.h"
+/*#include <curl/curl.h>*/
+
+/*input ninjo2dnudk.json and you will get out an array of windValues and a timestamp for last update*/
+windValue* ParseStringToWind(const char *input, int *lastUpdate)
+{
+    cJSON *json = cJSON_Parse(input);
+    if (json == NULL)
+    {
+        const char *error = cJSON_GetErrorPtr();
+        if (error != NULL)
+        {
+            printf("Error at: %s", error);
+        }
+        return 0;
+    }
+
+    cJSON *lastUpdateJson = cJSON_GetObjectItem(json, "lastupdate");
+    if (cJSON_IsString(lastUpdateJson) && (lastUpdateJson->valueint != 0))
+    {
+        printf("Input file last updated: \"%i\"\n", lastUpdateJson->valueint);
```

Comment:
Not sure why we are printing this?

Reply:
commented out for now. Still very relevant to read while the program is running